tools/snap/build_remote.py

```diff
@@ -115,7 +123,9 @@ def _build_snap(
             print(f'Build {target} for {",".join(archs)} (attempt {4-retry}/3) ended with '
                   f'exit code {exit_code}.')
 
-            failed_archs = [arch for arch in archs if status[target][arch] != 'Successfully built']
+            # This output may change, and is set by
+            # https://github.com/canonical/snapcraft/blob/8ab7fd0c8a1d3f13045bec41a6e0158c063faa9b/snapcraft/commands/remote.py#L278
```

Comment:
ah thanks for including the line, this was such a pain to track down last time

Reply:
Plus it moved when they put all the old code in the `legacy` folder, so it may in fact move again. But it's here for now :shrug: